basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/ElytronSslContextTestCase.java

```diff
@@ -0,0 +1,595 @@
+     * Validate edited attribute values in the model.
+     */
+    @Test
+    public void editClientSSLContextAttributesTest() throws Exception {
+        String clientSSLContextName = randomAlphanumeric(5), keyManagerName = randomAlphanumeric(5),
+                password = randomAlphanumeric(5), trustManagerName = randomAlphanumeric(5);
+        Integer sessionTimeoutValue = 999;
+        Address keyStoreAddress = createKeyStore(),
+                keyManagerAddress = elyOps.getElytronAddress(KEY_MANAGERS, keyManagerName),
+                clientSSLContextAddress = elyOps.getElytronAddress(CLIENT_SSL_CONTEXT, clientSSLContextName),
+                trustManagerAddress = elyOps.getElytronAddress(TRUST_MANAGERS, trustManagerName);
+        ModelNode credentialReferenceNode = new ModelNodePropertiesBuilder().addProperty(CLEAR_TEXT, password).build(),
+                protocolList = new ModelNodeListBuilder(new ModelNode(TLS_V11)).addNode(new ModelNode(TLS_V12))
+                        .build(),
+                newProtocolList = new ModelNodeListBuilder(new ModelNode(TLS_V12)).build();
+
+        try {
+            ops.add(keyManagerAddress, Values.of(ALGORITHM, ALGORITH_VALUE_1).and(KEY_STORE,
+                    keyStoreAddress.getLastPairValue()).and(CREDENTIAL_REFERENCE, credentialReferenceNode))
+                    .assertSuccess();
+            ops.add(clientSSLContextAddress, Values.of(KEY_MANAGERS, keyManagerName).and(PROTOCOLS, protocolList))
+                    .assertSuccess();
+            ops.add(trustManagerAddress, Values.of(ALGORITHM, ALGORITH_VALUE_1)
+                    .and(KEY_STORE, keyStoreAddress.getLastPairValue())).assertSuccess();
+
+            page.navigateToApplication().selectResource(CLIENT_SSL_CONTEXT_LABEL).getResourceManager()
+                    .selectByName(trustManagerName);
+            page.switchToConfigAreaTab(ATTRIBUTES_LABEL);
+
+            new ConfigChecker.Builder(client, clientSSLContextAddress).configFragment(page.getConfigFragment())
+                    .edit(TEXT, PROTOCOLS, TLS_V12)
+                    .edit(TEXT, TRUST_MANAGERS, trustManagerName)
+                    .edit(TEXT, SESSION_TIMEOUT, sessionTimeoutValue)
+                    .andSave().verifyFormSaved()
+                    .verifyAttribute(PROTOCOLS, newProtocolList)
+                    .verifyAttribute(TRUST_MANAGERS, trustManagerName)
+                    .verifyAttribute(SESSION_TIMEOUT, sessionTimeoutValue);
+        } finally {
+            ops.removeIfExists(clientSSLContextAddress);
+            ops.removeIfExists(trustManagerAddress);
+            ops.removeIfExists(keyManagerAddress);
+            ops.remove(keyStoreAddress);
+            adminOps.reloadIfRequired();
+        }
+    }
+
+    private Address createKeyStore() throws Exception {
+        String keyStoreName = randomAlphanumeric(5), password = randomAlphanumeric(5);
```

Comment:
I see your point but actually I have been using this approach for some time already and I am never confused when investigating issues thanks to console output.